bindings/c/CMakeLists.txt

```diff
@@ -52,14 +52,31 @@ if(NOT WIN32)
   if(OPEN_FOR_IDE)
     add_library(fdb_c_performance_test OBJECT test/performance_test.c test/test.h)
     add_library(fdb_c_ryw_benchmark OBJECT test/ryw_benchmark.c test/test.h)
+    add_library(mako OBJECT
```

Comment:
Can you please put the source files into a variable? Something like:

```
set(MAKO_SRCS
    test/mako/mako.c
    test/mako/mako.h
    test/mako/utils.c
    test/mako/utils.h
    test/mako/zipf.c
    test/mako/zipf.h)
```
and then when you add the library you can do

```
add_library(mako OBJECT ${MAKO_SRCS})
```

Reply:
Sure, good idea!  Thanks!